docker/docker-compose.yml

```diff
@@ -43,9 +44,12 @@ services:
 
     oms-events-static:
         restart: on-failure
-        image: aegee/oms-events-static:latest
+        image: aegee/nginx-static:latest
```

Comment:
this renaming is because the general idea is to habe only one nginx for all our services, right? And they'd be differentiated by a direct mount of the conf file

Reply:
Yes, I'm planning to do the same for statutory at least (here the images are also served by node, which isn't nice). We don't have image upload in other services now, only events and statutory, but we can use it for other services as well later (as this image is just nginx + curl).
Also we can even use the official image, but then we won't be able to add healthchecks